purchase_packaging_level_qty/models/purchase_order.py

```diff
@@ -0,0 +1,49 @@
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    total_transport_qty = fields.Char(
+        compute="_compute_total_transport_packaging_qty",
+    )
+
+    @api.depends("order_line.product_uom_qty", "order_line.product_id")
+    def _compute_total_transport_packaging_qty(self):
+        total_transport_packaging_level = self.env.company.purchase_packaging_level
+        for order in self:
+            total_transport_qty = sum(order.mapped("order_line.transport_qty"))
```

Comment:
the sum should be computed only if the feature is enabled

Reply:
Now the sum is computed only if `_is_purchase_transport_packaging_level_enabled()`